main/mariadb/APKBUILD

```diff
@@ -77,43 +81,39 @@ build() {
 		-DINSTALL_MYSQLSHAREDIR=share/mysql \
 		-DINSTALL_DOCDIR=share/mysql/docs \
 		-DINSTALL_SHAREDIR=share/mysql \
+		-DPLUGIN_ARCHIVE=YES \
+		-DPLUGIN_ARIA=YES \
+		-DPLUGIN_BLACKHOLE=YES \
+		-DPLUGIN_CSV=YES \
+		-DPLUGIN_MYISAM=YES \
+		-DPLUGIN_PARTITION=YES \
+		-DPLUGIN_TOKUDB=NO \
+		-DWITH_EMBEDDED_SERVER=ON \
+		-DWITH_EXTRA_CHARSETS=complex \
+		-DWITH_JEMALLOC=NO \
+		-DWITH_LIBARCHIVE=system \
+		-DWITH_LIBWRAP=OFF \
 		-DWITH_READLINE=ON \
-		-DWITH_ZLIB=system \
 		-DWITH_SSL=system \
-		-DWITH_LIBWRAP=OFF \
-		-DWITH_JEMALLOC=no \
-		-DWITH_EXTRA_CHARSETS=complex \
-		-DWITH_EMBEDDED_SERVER=ON \
-		-DWITH_ARCHIVE_STORAGE_ENGINE=1 \
-		-DWITH_BLACKHOLE_STORAGE_ENGINE=1 \
-		-DWITH_INNOBASE_STORAGE_ENGINE=1 \
-		-DWITH_PARTITION_STORAGE_ENGINE=1 \
-		-DPLUGIN_TOKUDB=NO \
-		-DWITHOUT_EXAMPLE_STORAGE_ENGINE=1 \
-		-DWITHOUT_FEDERATED_STORAGE_ENGINE=1 \
-		-DWITHOUT_PBXT_STORAGE_ENGINE=1 \
```

Comment:
Why have you removed these flags…?

Reply:
EXAMPLE, PBXT and FEDERATED have been droped quite white a while ago.
https://mariadb.com/kb/en/library/example-storage-engine/
https://mariadb.com/kb/en/library/pbxt-storage-engine/
https://mariadb.com/kb/en/library/federated-storage-engine/

Plus the -DWITHOUT_xxxxxx_STORAGE_ENGINE flag format is no longer in use, they all start with -DPLUGIN_xxxxx=NO